docs/whats-new.rst

```diff
@@ -4,7 +4,19 @@ What's New
 v0.1.4 (? ? 2015)
 --------------------
 
-Fixed bug in _calc_diagnostics. Now diagnostics start being averaged at
+Fixed `bug <https://github.com/pyqg/pyqg/issues/86>`__ related to the sign of advection terms.
```

Comment:
We have [extlinks](http://sphinx-doc.org/latest/ext/extlinks.html#confval-extlinks) enabled in sphinx, so you should be able to just do ```:issue:`86` ```